### DISCUSSION TITLE
cmus

### DISCUSSION THREAD

**Discussion by Sekki21956** [catppuccin/catppuccin#1954] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes

**Additional context 📷 (optional)**
Themeing for cmus.
Its done: https://github.com/Sekki21956/cmus
I wasn't able to use the real theme colors I had to convert them to xTerm colors via xterm256-color. I hope this still works.

**Comment by MagisterAdamus** [comment]
I made an alternative Catppuccin theme for cmus that works better with the Catppuccin CAVA visualizer themes. My theme is more “cotton-candyish” with pink/magenta and teal colors instead of green and orange. I also used boldface for the “titleline” and the “Artist / Album” and “Track” headings. The “code” is below:

```cfg
### 'Catppuccin' theme for CMus (for 255 color terms)
set color_cmdline_bg=default
set color_cmdline_fg=default
set color_error=211
set color_info=223
set color_separator=8
set color_statusline_bg=default
set color_statusline_fg=default
set color_titleline_bg=183
set color_titleline_fg=16
set color_titleline_attr=bold
set color_win_bg=default
set color_win_cur=117
set color_win_cur_sel_bg=117
set color_win_cur_sel_fg=235
set color_win_dir=default
set color_win_fg=default
set color_win_inactive_cur_sel_bg=0
set color_win_inactive_cur_sel_fg=default
set color_win_inactive_sel_bg=0
set color_win_inactive_sel_fg=default
set color_win_sel_bg=15
set color_win_sel_fg=235
set color_win_title_bg=default
set color_win_title_fg=183
set color_win_title_attr=bold
```

**Reply by shadow-absorber** [reply]
trying this out.... looks decent but the lines between different sections are harder to spot now but that might be fine

**Comment by shadow-absorber** [comment]
sooo how is this going with porting all of the flavours and accents???
